var/exp_repo/experiments/laghos/experiment.py

```diff
@@ -0,0 +1,74 @@
+from benchpark.directives import variant
```

Comment:
@pearce8 @august-knox We need to remove this file. I just kept it around for comparison